************* Module zipfile
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1:0: C0302: Too many lines in module (2450/1000) (too-many-lines)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:53:0: C0103: Class name "error" doesn't conform to PascalCase naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:82:0: C0103: Constant name "structEndArchive" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:83:0: C0103: Constant name "stringEndArchive" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:101:0: C0103: Constant name "structCentralDir" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:102:0: C0103: Constant name "stringCentralDir" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:128:0: C0103: Constant name "structFileHeader" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:129:0: C0103: Constant name "stringFileHeader" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:146:0: C0103: Constant name "structEndArchive64Locator" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:147:0: C0103: Constant name "stringEndArchive64Locator" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:152:0: C0103: Constant name "structEndArchive64" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:153:0: C0103: Constant name "stringEndArchive64" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:190:0: C0103: Argument name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:208:41: C0103: Variable name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:214:0: C0103: Function name "_EndRecData64" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:214:0: R0914: Too many local variables (17/15) (too-many-locals)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:240:9: C0103: Variable name "sz" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:228:17: W0612: Unused variable 'reloff' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:240:9: W0612: Unused variable 'sz' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:240:13: W0612: Unused variable 'create_version' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:240:29: W0612: Unused variable 'read_version' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:257:0: C0103: Function name "_EndRecData" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:294:4: C0103: Variable name "maxCommentStart" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:300:8: C0103: Variable name "recData" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:305:8: C0103: Variable name "commentSize" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1382:13: C0103: Attribute name "CRC" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:318:0: R0205: Class 'ZipInfo' inherits from object, can be safely removed from bases in python3 (useless-object-inheritance)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:318:0: R0902: Too many instance attributes (20/7) (too-many-instance-attributes)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:393:8: C0103: Variable name "hi" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:394:8: C0103: Variable name "lo" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:409:4: C0103: Method name "FileHeader" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:411:8: C0103: Variable name "dt" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:416:12: C0103: Variable name "CRC" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:418:12: C0103: Variable name "CRC" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:455:4: C0103: Method name "_encodeFilenameFlags" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:461:4: C0103: Method name "_decodeExtra" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:466:12: C0103: Variable name "tp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:466:16: C0103: Variable name "ln" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:505:20: W0612: Unused variable 'old' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:461:4: R0912: Too many branches (14/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:523:8: C0103: Variable name "st" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:489:20: W0201: Attribute 'file_size' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:542:12: W0201: Attribute 'file_size' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:545:12: W0201: Attribute 'file_size' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1382:37: W0201: Attribute 'file_size' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1814:8: W0201: Attribute 'file_size' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:497:20: W0201: Attribute 'compress_size' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1382:20: W0201: Attribute 'compress_size' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1745:12: W0201: Attribute 'compress_size' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:506:20: W0201: Attribute 'header_offset' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1379:12: W0201: Attribute 'header_offset' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1393:12: W0201: Attribute 'header_offset' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1762:16: W0201: Attribute 'header_offset' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1382:13: W0201: Attribute 'CRC' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1746:12: W0201: Attribute 'CRC' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1388:12: W0201: Attribute '_raw_time' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:558:0: C0103: Constant name "_crctable" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:560:8: W0612: Unused variable 'j' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:575:0: C0103: Function name "_ZipDecrypter" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:585:4: W0621: Redefining name 'crc32' from outer scope (line 24) (redefined-outer-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:580:4: C0103: Constant name "_crctable" doesn't conform to UPPER_CASE naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:580:4: W0603: Using the global statement (global-statement)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:585:4: C0103: Argument name "ch" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:589:4: C0103: Argument name "c" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:596:8: C0103: Variable name "p" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:603:12: C0103: Variable name "c" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:605:12: C0103: Variable name "c" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:619:16: W0212: Access to a protected member _encode_filter_properties of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:621:12: W0212: Access to a protected member _decode_filter_properties of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:653:16: W0212: Access to a protected member _decode_filter_properties of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:636:0: R0903: Too few public methods (1/2) (too-few-public-methods)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:704:4: R1705: Unnecessary "elif" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:721:4: R1705: Unnecessary "elif" after "return" (no-else-return)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:731:8: R1720: Unnecessary "else" after "raise" (no-else-raise)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:738:4: R0913: Too many arguments (6/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:757:4: C0103: Argument name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:777:8: C0103: Attribute name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:781:8: C0103: Variable name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:795:0: R0902: Too many instance attributes (21/7) (too-many-instance-attributes)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:809:4: W0231: __init__ method from base class 'BufferedIOBase' is not called (super-init-not-called)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:809:4: R0913: Too many arguments (6/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:855:12: C0103: Variable name "h" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:901:4: C0103: Argument name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:997:4: C0103: Argument name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1033:4: C0103: Argument name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1082:11: R1716: Simplify chained comparison between the operands (chained-comparison)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1115:4: W0231: __init__ method from base class 'BufferedIOBase' is not called (super-init-not-called)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1133:8: W0125: Using a conditional statement with a constant value (using-constant-test)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1145:8: W0125: Using a conditional statement with a constant value (using-constant-test)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1188:12: W0212: Access to a protected member _writing of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1227:8: C0103: Attribute name "_allowZip64" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1228:8: C0103: Attribute name "_didModify" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1230:8: C0103: Attribute name "NameToInfo" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1244:12: C0103: Attribute name "_filePassed" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1251:20: C0103: Attribute name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1262:8: C0103: Attribute name "_fileRefCnt" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1192:0: R0902: Too many instance attributes (19/7) (too-many-instance-attributes)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1218:4: R0913: Too many arguments (6/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1246:12: C0103: Variable name "modeDict" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1303:12: C0103: Variable name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1218:4: R0912: Too many branches (16/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1218:4: R0915: Too many statements (69/50) (too-many-statements)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1311:23: W0622: Redefining built-in 'type' (redefined-builtin)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1328:4: C0103: Method name "_RealGetContents" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1330:8: C0103: Variable name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1334:12: W0707: Consider explicitly re-raising using the 'from' keyword (raise-missing-from)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1356:8: C0103: Variable name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1376:12: C0103: Variable name "x" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1381:43: C0103: Variable name "t" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1381:46: C0103: Variable name "d" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1388:12: W0212: Access to a protected member _raw_time of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1392:12: W0212: Access to a protected member _decodeExtra of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1328:4: R0912: Too many branches (13/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1328:4: R0915: Too many statements (55/50) (too-many-statements)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1431:55: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1466:12: C0415: Import outside toplevel (warnings) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1475:42: C0103: Variable name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1511:12: W0212: Access to a protected member _compresslevel of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1478:4: R0912: Too many branches (22/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1478:4: R0915: Too many statements (53/50) (too-many-statements)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1709:12: C0415: Import outside toplevel (warnings) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1754:16: W0212: Access to a protected member _compresslevel of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1756:16: W0212: Access to a protected member _compresslevel of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1791:12: W0212: Access to a protected member _compresslevel of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1812:12: W0212: Access to a protected member _compresslevel of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1841:12: C0103: Variable name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1845:4: R0914: Too many local variables (24/15) (too-many-locals)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1847:12: C0103: Variable name "dt" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1886:38: W0212: Access to a protected member _encodeFilenameFlags of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1911:8: C0103: Variable name "centDirCount" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1912:8: C0103: Variable name "centDirSize" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1913:8: C0103: Variable name "centDirOffset" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1936:12: C0103: Variable name "centDirCount" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1937:12: C0103: Variable name "centDirSize" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1938:12: C0103: Variable name "centDirOffset" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1845:4: R0912: Too many branches (14/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1845:4: R0915: Too many statements (65/50) (too-many-statements)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1947:4: C0103: Argument name "fp" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1957:4: R0913: Too many arguments (6/5) (too-many-arguments)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1983:8: W0622: Redefining built-in 'dir' (redefined-builtin)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1984:8: R1702: Too many nested blocks (6/5) (too-many-nested-blocks)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1984:8: R1702: Too many nested blocks (6/5) (too-many-nested-blocks)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1983:8: W0612: Unused variable 'dir' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2003:20: W0612: Unused variable 'root' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1963:4: R0912: Too many branches (25/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:1963:4: R0915: Too many statements (57/50) (too-many-statements)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2053:12: C0415: Import outside toplevel (py_compile) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2045:4: R0912: Too many branches (19/12) (too-many-branches)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2045:4: R0915: Too many statements (54/50) (too-many-statements)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2184:14: W0612: Unused variable 'tail' (unused-variable)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2205:16: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2235:12: W0642: Invalid assignment to cls in method (self-cls-assignment)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2250:23: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2256:24: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2249:19: E0203: Access to member '__names' before its definition line 2250 (access-member-before-definition)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2255:19: E0203: Access to member '__lookup' before its definition line 2256 (access-member-before-definition)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2250:8: W0201: Attribute '__names' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2256:8: W0201: Attribute '__lookup' defined outside __init__ (attribute-defined-outside-init)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2329:8: C0103: Attribute name "at" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2350:4: C0103: Argument name "at" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2360:26: W0212: Access to a protected member _name_set of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2375:8: W0622: Redefining built-in 'next' (redefined-builtin)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2389:4: C0415: Import outside toplevel (argparse) (import-outside-toplevel)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2408:34: C0103: Variable name "zf" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2416:34: C0103: Variable name "zf" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2421:34: C0103: Variable name "zf" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2428:8: C0103: Function name "addToZip" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2428:8: C0103: Argument name "zf" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2434:20: C0103: Variable name "nm" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/zipfile.py:2439:39: C0103: Variable name "zf" doesn't conform to snake_case naming style (invalid-name)

-----------------------------------
Your code has been rated at 8.81/10

